pytest_logger/plugin.py

```diff
@@ -14,6 +15,16 @@
     string_type = str
 
 
+@pytest.hookimpl
+def pytest_addoption(parser):
+    """Add options to control logger"""
+    parser.addini(
+        name='logger_logdir',
+        help='base directory with log files for file loggers [basetemp]',
+        default=None,
+    )
+    parser.getgroup('logger').addoption('--logger-logdir', dest='logger_logdir')
```

Comment:
OK. I am not sure if `logger` prefix is optimal, but is aligned with this plugin.